libs/designsystem/src/lib/directives/size/size.directive.ts

```diff
@@ -1,30 +1,13 @@
-import { Directive, Input, HostBinding } from '@angular/core';
+import { Directive } from '@angular/core';
 
 @Directive({
-  // don't worry. I know what i am doing!
   // tslint:disable-next-line:directive-selector
-  selector: 'button[size], kirby-icon[size], kirby-avatar[size], kirby-item[size]',
+  selector: ' ',
```

Comment:
This won't make the size directive apply to everything? 🤔 
Maybe:
```suggestion
  selector: '[kirbyDeprecatedSizeDirectiveDontUse]',
```

Reply:
06d4b7f